_dashboards/visualize/viz-index.md

```diff
@@ -0,0 +1,114 @@
+
+## Heat maps
+
+<img src="{{site.url}}{{site.baseurl}}/images/heatmap-1.png" alt="Example heat map in OpenSearch Dashboards" width="60%" height="30%">
+
+A heat map is a view of a histogram (a graphical representation of the distribution of numerical data) over time. Instead of using bar height as a representation of frequency, as with a histogram, heat maps use cells, coloring a cell proportional to the number of values. Learn how to create and use [Heat maps]({{site.url}}{{site.baseurl}}/dashboards/visualize/heat-maps) in Dashboards.
+## Line graphs
+
+<img src="{{site.url}}{{site.baseurl}}/images/line-graph-1.png" alt="Example line graph in OpenSearch Dashboards" width="60%" height="30%">
+
+Line charts compare changes in measure values over period of time, such as gross sales by month or gross sales and net sales by month. Learn how to create and use [Line charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/line-charts) in Dashboards.
+
+## Metric charts
+
+<img src="{{site.url}}{{site.baseurl}}/images/metric-chart-1.png" alt="Example metric chart in OpenSearch Dashboards" width="60%" height="30%">
+
+Metric charts show a numerical value, such as a key performance indicator (KPI), to visualize a comparison between a key value and its target value. Metric charts display a value comparison, the two values being compared, and a progress bar. Learn how to create and use [Metric charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/metric-charts) in Dashboards.
+
+## Maps
+### Coordinate maps
+
+<img src="{{site.url}}{{site.baseurl}}/images/map-1.png" alt="Example map in OpenSearch Dashboards" width="60%" height="30%">
+
+Coordinate maps show location-based data on a map. Use coordinate maps to visualize GPS data (latitude and longitude coordinates) on a map.Learn how to create and use [Coordinate maps]({{site.url}}{{site.baseurl}}/dashboards/visualize/coordinate-maps) in Dashboards.
+
+For information about OpenSearch-supported coordinate field types, see [Geographic field types]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-shape/) and [Cartesian field types]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/xy/). 
+
+### Region maps
+
+Region maps show patterns and trends across geographic locations. A region map is one of the basemaps in Dashboards. Learn how to create and use [Region maps]({{site.url}}{{site.baseurl}}/dashboards/visualize/geojson-regionmaps/) in Dashboards. 
+
+## Pie charts
+
+<img src="{{site.url}}{{site.baseurl}}/images/pie-chart-1.png" alt="Example pie chart in OpenSearch Dashboards" width="60%" height="30%">
+
+Pie charts compare values for items in a dimension, such as  a percentage of a total amount. Learn how to create and use [Pie charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/pie-charts/) in Dashboards.
```

Comment:
Should this be something like "Reference [Pie charts]({{site.url}}{{site.baseurl}}/dashboards/visualize/pie-charts/) to learn how to create and use Pie charts in Dashboards." ?